telemetry/definitions/commonDefinitions.json

```diff
@@ -570,6 +570,26 @@
             "type": "int",
             "description": "The number of import statements included with recommendation."
         },
+        {
+            "name": "codewhispererSupplementalContextTimeout",
+            "type": "boolean",
+            "description": "If the supplemental context fetching timeout or not"
+        },
+        {
+            "name": "codewhispererSupplementalContextIsUtg",
+            "type": "boolean",
+            "description": "If the supplemental context is for test file(UTG) or src file"
+        },
+        {
+            "name": "codewhispererSupplementalContextLatency",
+            "type": "int",
```

Comment:
should this be a double like other latency?

Reply:
okk sg